doc/src/tutorial.xml

```diff
@@ -280,8 +280,8 @@ top/
       in their requirements, too.  Of course, any project can add
       include paths to those specified by its parents. <footnote>
       <para>Many
-      features will be overridden,
-      rather than added-to, in subprojects.  See <xref
+      features will be overridden in,
+      rather than added-to, subprojects.  See <xref
```

Comment:
It's perfectly valid in the main clause to say:
`Many features will be overridden in subprojects.`
but saying in the alternative clause:
`Many features will be added-to in subprojects`
is not as correct as saying:
`Many features will be added-to subprojects.`

If you do not see this I will change it back and not try to correct it.